### ISSUE TITLE
Arctos Report Request: Summary transaction information

### ISSUE THREAD

**Issue by happiah-madson** [ArctosDB/arctos#6941]
**Description of the report** - (Apologies, this may not be a report request, but maybe it is?) In our old database, we have summary information about what is in an accession:
<img width="1093" alt="Screen Shot 2023-11-08 at 4 47 38 PM" src="https://github.com/ArctosDB/arctos/assets/47900639/3ae57daf-167c-474b-9141-4286c277b478">

I'm wondering if it would be possible for us to be able to see this information in Arctos, as well?

**Collection for which the report is needed** - OGL:Genomic

**Priority** - Medium

**Comment by dustymc** [comment]
Yep, job for a report. Seems like a variation of https://github.com/ArctosDB/arctos/issues/6940, is that doing what you want?

**Comment by happiah-madson** [comment]
Yep, it is definitely a variation on #6940 but without the date constraints. Once I figure out how to make that work I'll let you know if it's doing what I want...
